packages/manager/apps/carrier-sip/package.json

```diff
@@ -0,0 +1,64 @@
+{
+  "name": "@ovh-ux/manager-carrier-sip-app",
+  "version": "0.0.0",
+  "private": true,
+  "license": "BSD-3-Clause",
+  "main": "./src/index.js",
+  "scripts": {
+    "build": "webpack --env.production",
+    "dev": "webpack-dev-server",
+    "dev:watch": "yarn run dev",
+    "start": "lerna exec --stream --scope='@ovh-ux/manager-carrier-sip-app' --include-filtered-dependencies -- npm run build --if-present",
+    "start:dev": "lerna exec --stream --scope='@ovh-ux/manager-carrier-sip-app' --include-filtered-dependencies -- npm run dev --if-present",
+    "start:watch": "lerna exec --stream --parallel --scope='@ovh-ux/manager-carrier-sip-app' --include-filtered-dependencies -- npm run dev:watch --if-present"
+  },
+  "dependencies": {
+    "@ovh-ux/manager-carrier-sip": "^0.0.0",
+    "@ovh-ux/manager-config": "^0.3.0",
+    "@ovh-ux/manager-core": "^7.1.2",
+    "@ovh-ux/manager-telecom-styles": "^3.1.0",
+    "@ovh-ux/ng-ovh-api-wrappers": "^4.0.4",
+    "@ovh-ux/ng-ovh-checkbox-table": "^1.0.2",
+    "@ovh-ux/ng-ovh-http": "^4.0.4",
+    "@ovh-ux/ng-ovh-mondial-relay": "^6.0.1",
+    "@ovh-ux/ng-ovh-request-tagger": "^1.0.0",
+    "@ovh-ux/ng-ovh-sso-auth": "^4.2.2",
+    "@ovh-ux/ng-ovh-swimming-poll": "^4.0.0",
+    "@ovh-ux/ng-ovh-telecom-universe-components": "^4.1.2",
+    "@ovh-ux/ng-translate-async-loader": "^2.0.2",
+    "@uirouter/angularjs": "^1.0.22",
+    "CSV-JS": "^1.2.0",
+    "angular": "^1.7.8",
+    "angular-aria": "^1.7.5",
+    "angular-cookies": "^1.7.8",
+    "angular-dynamic-locale": "^0.1.37",
+    "angular-i18n": "^1.7.5",
+    "angular-leaflet-directive": "0.8.5",
+    "angular-resource": "^1.7.5",
+    "angular-sanitize": "^1.7.5",
+    "angular-translate": "^2.18.1",
+    "angular-translate-loader-pluggable": "^1.3.1",
+    "angular-ui-bootstrap": "~1.3.3",
+    "angular-validation-match": "^1.9.0",
+    "chart.js": "^2.8.0",
+    "flatpickr": "^4.5.2",
+    "ip-address": "^5.9.2",
+    "jquery": "^3.4.1",
+    "jsplumb": "^2.11.2",
```

Comment:
😭

Reply:
Theses dependencies were defined as `peerDependecy` on `ngOvhTelecomUniverseComponents` module so that's why I have to add them.